TASK.md

```diff
@@ -0,0 +1,28 @@
+# Task Definition Template
+
+## Title: [descriptive task name]
+
+
+### About:
+- Level: [L1 | L2 |L3]
+- Repo URL: http://github.com/...
+    - Make sure the repo has an appropriate README with instructions for running the code
+- Language: [python]
+- Operating System(s): [Linux, Mac, Windows]
+- Primary contact email: e.g. carlosg@uw.edu
+- Additional dependencies. E.g. 
+    - docker
+    - GPU   
+    - AWS/Azure/GCP acccount
+
+## Description
+
+A description of the work to perform. Add enough details for someone unfamiliar with the project to work on the task. For L2 and L3 tasks, breaking them up in 2-3 steps will be helpful.
+
+## Tags: 
+
+Apply appropriate tags:
+- Level: L1, L2, L3
+- Language: python, shell, typescript
+- Cloud: aws, azure, gcp
+ 
```

Comment:
Let's convert this to an issue template: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository